.github/workflows/workspace.yaml

```diff
@@ -10,30 +10,30 @@ env:
 
 jobs:
   rustfmt:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Linux, large]
+    container: mobilecoin/rust-sgx-base:latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
-        with:
-          toolchain: nightly
-          components: rustfmt
-          override: true
-          profile: minimal
+      # actions-rs/toolchain does not support the newer toml file format
+      # see https://github.com/actions-rs/toolchain/issues/126
+      - name: Setup rust toolchain
+        run: rustup show
+        working-directory: ./trusted
```

Comment:
Consider using [`defaults.run`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#defaultsrun) to default the `working-directory` for this workflow